glossary.md

```diff
@@ -39,3 +39,19 @@ USA -- typeOf ------------> Country
 USA -- name --------------> United States of America
 USA -- containedInPlace --> northamerica
 ```
+
+- **Measurement method**
+
+The source of and technique used for measuring a statistical variable. Describes how a measurement is made, whether by count or estimate or some other approach, and names the group making the measurement. Examples include `BosniaCensus` and `WorldHealthOrganizationEstimates`.
+
+- **Observation period**
+
+The time period over which an observation is made.
+
+- **Unit**
+
+The unit of measurement. Examples include Kelvin, Celsius, inches, light years, and slugs.
+
+- **Scaling factor**
+
+Factor by which a measurement is adjusted to fit a certain format. For example, a proportion of 0.05 displayed as 5% has a scaling factor of 100.
```

Comment:
After I referenced the description from the browser, I think we can explain this a bit more similarly to what's there.

I think first, we should say that this is a property of StatisticalVariables, used when a StatisticalVariable has a measurementDenominator. If the stat is not a 1 to 1 proportion, but a 1:N like 1:100 for percent, then the scalingFactor describes the N multiplier for the denominator. (Ofc what I just wrote is very technical and should probably be English-ed

Reply:
Hi @tjann, I updated the definition to include language around statistical variables and clarifying the example a bit (?). Please let me know if you have any additional comments!